Query

tingamapuro04 opened the pull request:
I have worked on the query branch where I have included many queries for the animals table in the vet_clic database.
I have updated the queries.sql with the various queries required.
Below I am attaching the screenshots of the various results.
I have used Visual studio for the work and not the terminal.

<img width="439" alt="average" src="https://user-images.githubusercontent.com/82979239/186243336-cd6ce5d7-2718-4486-b1b2-f31c6d34e44e.png">
<img width="720" alt="delete" src="https://user-images.githubusercontent.com/82979239/186242668-a91869f0-1126-46b3-9daf-910618463dea.png">
<img width="443" alt="escape" src="https://user-images.githubusercontent.com/82979239/186242819-048f44fb-6ac4-4e6a-afdb-494c1973ccac.png">
<img width="439" alt="groupBy" src="https://user-images.githubusercontent.com/82979239/186242859-73d119f7-3e77-48d8-a9b5-9cf9c7bad74f.png">
<img width="632" alt="pokemon" src="https://user-images.githubusercontent.com/82979239/186242941-ddb5038a-fff0-4e60-815d-2acdc18e744d.png">
<img width="717" alt="savepoint" src="https://user-images.githubusercontent.com/82979239/186243008-eec7fb56-2152-4076-804e-f2482a0cc7ab.png">
<img width="601" alt="species_avg" src="https://user-images.githubusercontent.com/82979239/186243079-9eadca73-0b93-4dbd-b275-8caf3bc12b38.png">
<img width="441" alt="total" src="https://user-images.githubusercontent.com/82979239/186243149-c2abc15d-ea84-4185-93e8-e3a2b9068a3c.png">

tingamapuro04 committed:
Update schema.sql

Update schema.sql

tingamapuro04 committed:
Update data.sql

Update data.sql

tingamapuro04 committed:
Update queries.sql

Update queries.sql

tingamapuro04 committed:
Update queries.sql

Update queries.sql

tingamapuro04 committed:
Update queries.sql

Update queries.sql

tingamapuro04 committed:
Update queries.sql

Update queries.sql

mateo951 commented on schema.sql:
```diff
@@ -6,5 +6,6 @@ CREATE TABLE animals (
   neutered BOOL,
-  weight_kg DECIMAL(4,2)
+  weight_kg DECIMAL(4,2),
+  species TEXT
```
Rather than editing your preexisting table, it's important that you use the ALTER method in order to practice SQL Statements 🤓

mateo951 commented on data.sql:
```diff
@@ -1,2 +1,10 @@
 /* Populate database with sample data. */
 INSERT INTO animals VALUES (1, 'Agumon', '2020-02-03', 0, '1', 10.23), (2, 'Gabumon', '2018-11-15', 2, '1', 08.00), (3, 'Pikachu', '2021-01-07', 1, '0', 15.04), (4, 'Devimon', '2017-05-12', 5, '1', 11.00);
+
+INSERT INTO animals VALUES
+(6, 'Plantmon', '2021-11-15', 2, '1', -05.70, NULL),
+(7, 'Squirtle','1993-04-02', 3, '0', -12.13, NULL),
+(8, 'Angemon', '2005-06-12', 1, '1', -45, NULL),
+(9, 'Boarmon', '2005-06-07', 7, '1', 20.40, NULL),
+(10, 'Blossom', '1998-10-13', 3, '1', 17.00, NULL),
+(11, 'Ditto', '2022-05-14', 4, '1', 22.00, NULL);
```
Propper values were inserted✔ Good job!

mateo951 commented on queries.sql:
```diff
@@ -8,3 +8,23 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.50;
 SELECT * FROM animals WHERE neutered <> '0';
 SELECT * FROM animals WHERE name <> 'Gabumon';
 SELECT * FROM animals WHERE weight_kg >= 10.40 AND weight_kg <= 17.30;
+
```
I know you probably did the transactions on your terminal but it's important that you add them 🤓 Please include them so that your project meets the requirements 😎

mateo951 reviewed (approved):
Hi @tingamapuro04, 

🔥 Highlights 🔥
- Queries Answered ✔
- Data Introduced ✔

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

tingamapuro04 committed:
Update schema.sql

Add a query for adding a column

tingamapuro04 committed:
Add the first Transaction

Add a transaction for updating the species column

tingamapuro04 committed:
Update queries.sql

tingamapuro04 committed:
Update queries.sql

Add queries for updating weight_kg

tingamapuro04 commented:
As to the suggestions I have Added the transactions that I implemented in the queries.sql. Also instead of modifying the preexisting query, I have updated it using the ALTER  query to add the species column

prolajumokeoni reviewed (approved):
Hi @tingamapuro04, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- No linter errors ✅ 
- Git flow was used correctly 🚀 
- Good design 💯 
- Good Pr title and description 💪🏿 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
